sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/tests/Azure.Monitor.OpenTelemetry.AspNetCore.Integration.Tests/TelemetryValidationHelper.cs

```diff
@@ -71,37 +75,177 @@ private static void ValidateProperties(string description, string jsonString, Li
 #endif
         }
 
+        /*
+         * Notes on field validation:
+         * Remember that this test will be run as both a Recording and Live.
+         * We can't include unique ids becasue they are unique per test run (ie: Id, OperationId, ParentId, etc).
+         * We can't include timing fields because would be unique per test run(ie: TimeGenerated, DurationMS, PerformanceBucket).
+         * We can't include client fields because we can't control where these tests run (ie: ClientOS, ClientCity, ClientCountryOrRegion, etc).
+         */
+
         public struct ExpectedAppDependency
         {
+            //public string TenantId { get; set; }
+            //public string TimeGenerated { get; set; }
+            //public string Id { get; set; }
```

Comment:
I've left all available fields commented out to show what is available.
During review, we can have a conversation about including any of these in the test.
I'm planning to remove the commented fields before merging this PR.